ios/scaffold/scaffold.entitlements

```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>application-identifier</key>
```

Comment:
explanation for this is on the Astro PR: https://github.com/mobify/astro/pull/614